lib/govuk_index/publishing_event_processor.rb

```diff
@@ -0,0 +1,46 @@
+require 'govuk_index/elasticsearch_presenter'
+
+module GovukIndex
```

Comment:
Perhaps it would be good to separate out the indexing from the rabbitmq processing? This class mixes concerns a bit. Separating them would make the tests cleaner too probably.